README.md

```diff
@@ -184,6 +184,41 @@ end
 See [eight_bit_color.rb](lib/super_diff/csi/eight_bit_color.rb)
 for the list of available colors.
 
+### Enable or disable the key
+
+You can disable the key by changing the following config (default: true)
+
+``` ruby
+SuperDiff.configure do |config|
+  config.key_enabled = false
+end
+```
+
+### Elision
```

Comment:
Perhaps for consistency with other headers (and to explain the concept better) what if we say:

``` suggestion
### Hiding unimportant lines
```

Reply:
Good call. I'd be lying if I said I didn't have to google "Elision" 😂